Game/MyProject/IvGraphics/OGL/IvFrameBufferOGL.h

```diff
@@ -0,0 +1,44 @@
+//
+//  IvFrameBufferOGL.hpp
+//  IvGraphics
+//
+//  Created by Monica Olanescu - (p) on 4/4/19.
+//
+#pragma once
+
+#include"IvFrameBuffer.h"
+
+#if defined(__APPLE__) && defined(__MACH__)
+#include <OpenGL/gl3.h>
+#else
+#include <GL/glew.h>
+#include <GL/gl.h>
+#endif
+
+//-------------------------------------------------------------------------------
+//-- Typedefs, Structs ----------------------------------------------------------
+//-------------------------------------------------------------------------------
+
+//-------------------------------------------------------------------------------
+//-- Classes --------------------------------------------------------------------
+//-------------------------------------------------------------------------------
+class IvFrameBufferOGL:private IvFrameBuffer
+{
+public:
+    friend class IvResourceManagerOGL;
+    
+    void Bind(int width=1280,int height=720);
+    bool Unbind();
+    
+    std::vector<IvRenderTarget*> GetTextures() const ;
```

Comment:
```suggestion
    const std::vector<IvRenderTarget*>& GetTextures() const ;
```
To avoid vector copy